app/controllers/posts_controller.rb

```diff
@@ -7,4 +7,26 @@ def index
   def show
     @post = Post.find(params[:id])
   end
+
+  def new
+    @post = Post.new
+  end
+
+  def create
+    author = User.find(params[:user_id])
```

Comment:
As mentioned in the requirements, posts are supposed to be created by the user returned by the `current_user` method which you created in the `application_controller.rb` file